notanorm/base.py

```diff
@@ -315,6 +325,11 @@ def __resolve_on_field(self, k, tab):
         return k
 
 
+class Where:
+    def __init__(self, ops: List[Dict[str, Union[Op, SubQ]]]):
```

Comment:
Also, since we seem (?) to support list values for `IN`, should the union be `Union[Op, SubQ, List[str]]`?

Reply:
yep

Reply:
actually it's a list of any valid query values